frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -80,6 +83,43 @@ const ConsolePluginStatus: React.FC<ConsolePluginStatusType> = ({ enabled, plugi
 
 const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
   const { t } = useTranslation();
+  const toastContext = useToast();
+  const [pluginsData, pluginsError] = useURLPoll<ConsolePluginsListResponseType>(
+    `${window.SERVER_FLAGS.basePath}api/plugins`,
+  );
+
+  const prevPluginsDataRef = React.useRef<ConsolePluginsListResponseType>();
+  React.useEffect(() => {
+    prevPluginsDataRef.current = pluginsData;
+  });
+  const prevPluginsData = prevPluginsDataRef.current;
+
+  const pluginsStateInitialized = _.isEmpty(pluginsError) && !_.isEmpty(prevPluginsData);
+  const pluginsChanged = !_.isEqual(prevPluginsData?.plugins, pluginsData?.plugins);
+  const consoleCommitChanged = !_.isEqual(
+    prevPluginsData?.consoleCommit,
+    pluginsData?.consoleCommit,
+  );
```

Comment:
For strings, I'd suggest simply

```suggestion
  const consoleCommitChanged = prevPluginsData?.consoleCommit !== pluginsData?.consoleCommit;
```